package.json

```diff
@@ -1,20 +1,13 @@
 {
-  "name": "create-copilot-app",
-  "description": "A custom app starter kit for Copilot",
-  "version": "2.0.0",
+  "name": "task-app",
+  "description": "A comprehensive task management app",
+  "version": "1.0.0",
```

Comment:
Might make more sense to make it `0.1.0` since we don't have a stable release yet